torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -305,6 +305,10 @@ class InsertObserversHelper {
       bool is_entry_point = false,
       bool is_user_defined_function = false);
 
+  // Record v as "ready for observation" by storing it in values_to_observe and
+  // block_observed_values. If v is a part of a delayed observation pattern,
+  // record v's descendant (per delay rules) instead. The observers are inserted
+  // at a later stage by reading the state created by this function.
   void recordObserved(
```

Comment:
value_to_observe stores the values that we'll need to insert observers for, block_observed_values are the values already observed in the current block